packages/vitest/src/runtime/worker.ts

```diff
@@ -11,10 +11,25 @@ import { disposeInternalListeners } from './workers/utils'
 
 if (isChildProcess()) {
   setProcessTitle(`vitest ${poolId}`)
+
+  const isProfiling = process.execArgv.filter(
```

Comment:
Did you mean `.some`? Filter always returns an array which is truthy

Reply:
Good catch! Nice to see that even if this work-around is always applied, our tests still pass.